src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryPipelineSpark.java

```diff
@@ -40,19 +40,56 @@
 import static org.broadinstitute.hellbender.tools.spark.sv.StructuralVariationDiscoveryArgumentCollection.FindBreakpointEvidenceSparkArgumentCollection;
 
 /**
- * Tool to run the sv pipeline up to and including variant discovery.
- * Expected input is a BAM with around 30x coverage.  Coverage much lower than that probably won't work well.
+ * Master tool to run the structural variation discovery pipeline.
+ *
+ * <p>Runs FindBreakpointEvidenceSpark and DiscoverVariantsFromContigAlignmentsSAMSpark.</p>
```

Comment:
not sure exactly what the style guide on this is, but for `FindBreakpointEvidenceSpark` and `DiscoverVariantsFromContigAlignmentsSAMSpark` there's the `{@link symbolName}`, just incase something changes in the future.